datasets/xglue/xglue.py

```diff
@@ -0,0 +1,526 @@
+                It covers 3 languages, including English, French
+                and German. Each labeled instance is a 3-tuple:
+                <question, passage, label>. The label indicates
+                whether the passage is the answer of the question
+                (1), or not (0). This dataset is constructed  based on
+                a commercial search engine. Accuracy (ACC) of
+                the binary classification should be used as the metric.
+            """
+            ),
+            data_dir="QAM",
+            citation="",
+            url="",
+        ),
+        XGlueConfig(
+            name="qg",
+            description=textwrap.dedent(
+                """\
+                The Question Generation (QG) task aims to
+generate a question for a given passage. <passage, question> pairs were collected from a commercial search engine. It covers 6 languages, including English, French, German, Spanish, Italian and
+Portuguese. BLEU-4 score should be used as the metric.
+            """
+            ),
+            data_dir="QG",
+            citation="",
+            url="",
+        ),
+        XGlueConfig(
+            name="ntg",
+            description=textwrap.dedent(
+                """\
+                News Title Generation (NTG) task aims
+                to generate a proper title for a given news body.
+                We collect <news body, news title> pairs from a
+                commercial news website. It covers 5 languages,
+                including German, English, French, Spanish and
+                Russian. BLEU-4 score should be used as the metric.
+            """
+            ),
+            data_dir="NTG",
+            citation="",
+            url="",
+        ),
+    ]
+
+    def _info(self):
+        if self.config.name == "ner":
+            features = {
+                "words": datasets.Sequence(datasets.Value("string")),
+                "ner": datasets.Sequence(datasets.Value("string")),
+            }
+        elif self.config.name == "pos":
+            features = {
+                "words": datasets.Sequence(datasets.Value("string")),
+                "pos": datasets.Sequence(datasets.Value("string")),
+            }
+        elif self.config.name == "mlqa":
+            features = {
+                "context": datasets.Value("string"),
+                "question": datasets.Value("string"),
+                "answers": datasets.features.Sequence(
+                    {"answer_start": datasets.Value("int32"), "text": datasets.Value("string")}
+                ),
+                # These are the features of your dataset like images, labels ...
+            }
+        elif self.config.name == "nc":
+            features = {
+                "news_title": datasets.Value("string"),
+                "news_body": datasets.Value("string"),
+                "news_category": datasets.Value("string"),
```

Comment:
is is possible to use ClassLabel here ?

Reply:
> Awesome good job !
> 
> I left a few comments about feature naming and types.
> Also It looks like the dummy data files are quite heavy (450KB each). It should be possible to remove some data files inside the zip files to make them lighter (for example remove the mlqa files from the dummy_data.zip for for ner)

Tried to reduce as much as possible. It total now around ~2MB. Think that's fine given that we have 11 configs a 19 splits

Reply:
Thank you so much ! This makes cloning the library faster